### ISSUE TITLE
[BITV] 9.2.1.1/3.1 - It is not possible to focus and activate the "Set expiration date" input field using the keyboard. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36979]
- [ ] Fix
- [ ] Backport

![06f36c7dd756677b96aaad1ad81a9f93](https://user-images.githubusercontent.com/1315170/222244817-6d987ecb-dbfb-45fe-ac33-138571645e6d.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v3-n1

</details>

- [ ] 9.2.1.1/10.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v10-n1

**Comment by susnux** [comment]
I tried to reproduce this issue, but it works for me on NC25, NC26 and NC27. Do you known which Nextcloud version was tested?

https://user-images.githubusercontent.com/1855448/235859110-cdcba930-9d2d-4aea-ae57-9e135d55ad7e.mp4

**Comment by JuliaKirschenheuter** [comment]
should be fine: https://github.com/nextcloud/server/issues/33914

**Comment by JuliaKirschenheuter** [comment]
closing issue
